B/boost/build_tarballs.jl

```diff
@@ -44,12 +44,17 @@ if [[ $target == *apple* ]]; then
         # See https://github.com/boostorg/context/issues/170#issuecomment-863669877
         extraargs="abi=aapcs ${extraargs}"
     fi
-elif [[ $target == x86_64*mingw* ]]; then
+elif [[ $target == *mingw* ]]; then
     targetos=windows
-    extraargs="address-model=64 define=_WIN32_WINNT=0x0602 binary-format=pe abi=ms link=shared"
-elif [[ $target == i686*mingw* ]]; then
-    targetos=windows
-    extraargs="address-model=32 define=_WIN32_WINNT=0x0602 binary-format=pe abi=ms link=shared"
+    echo "using gcc : 10.2 : $CXX : <cxxflags>\\"-DBOOST_USE_WINAPI_VERSION=0x0601 -D_WIN32_WINNT=0x0601 -DBOOST_STACKTRACE_USE_WINDBG=1\\" <linkflags>\\"-DBOOST_STACKTRACE_USE_WINDBG=1 -ldl -lstdc++\\" ;" > project-config.jam
```

Comment:
Why do you need to set `_WIN32_WINNT`?  We already set it to to `0x0A00` (Window 10): https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/087e70545de164cad9b92d459f88b7b275988dca/src/Runner.jl#L538

Reply:
Windows 10 builds fail with errors